src/main/java/com/github/psambit9791/jdsp/windows/_Window.java

```diff
@@ -57,9 +63,30 @@ public int extend(int length, boolean sym) {
         }
     }
 
-//    /**
-//     * This method computes the window and returns it
-//     * @return double[] The generated window
-//     */
-//    public abstract double[] getWindow();
+    /**
+     * This method computes the window and returns it
+     * @return double[] The generated window
+     */
+    public abstract double[] getWindow();
```

Comment:
I have updated UtilMethods to have this function in a previous commit. I will also introduce the applyWindow() under UtilMethods. Will keep the Window module clean.

Reasoning: Applying a window to a signal is an utility. The window itself has to exist as its own object though.

Reply:
From a programmer perspective, it doesn't make much sense to put that in the UtilMethods for three reasons:

1. The functions getWindow and applyWindow are functions that are inherent to a _Window-class object.
2. Otherwise the _Window-function doesn't have much use, other than some very basic stuff
3. You are cluttering the UtilMethods with too much methods that come from too many places; this can become confusing over time

I would strongly vouch to keep things organized in their rightful class and keep getWindow and applyWindow in the _Window-class